lib/prometheus/business_collector.rb

```diff
@@ -122,6 +128,26 @@ def collect
     @client_50plus_policies.observe(
       policies_50plus_hosts.where('policies.account_id' => client_accounts.select(:id)).count.size
     )
+
+    @total_reports_per_month.observe(
+      TestResult.where(
+        'updated_at >= ? AND updated_at <= ?',
```

Comment:
What's the rationale behind using `updated_at` vs. `created_at`? Wouldn't the `updated_at` break the "Counter requirement" promise? :slightly_smiling_face: